resources/kcp/charts/kyma-metrics-collector/templates/public-cloud-info-config-map.yaml

```diff
@@ -357,7 +357,7 @@ data:
          "memory": 320
        }
      },
-      "sap-converged-cloud": {
+      "openstack": {
```

Comment:
Why the `sap-converged-cloud` value changed?

Reply:
It was added in [PR](https://github.com/kyma-project/control-plane/pull/3247). But it is incorrect.